src/@next/IndexTable/IndexTable.tsx

```diff
@@ -8,14 +8,23 @@ import { Checkbox } from '../Checkbox';
 import { CheckboxProps } from '../Checkbox';
 import { Row } from './components/Row/Row';
 import { StyledIndexTable } from './IndexTableStyle';
+import { LoadingState } from './components/LoadingState';
+
+export interface CustomIndexTableProps
+  extends Omit<IndexTableProps, 'emptySearchTitle'> {
+  emptySearchTitle?: string;
```

Comment:
It should be optional because, now we can use emptyState prop to render empty table.